hub/api/dataset_utils.py

```diff
@@ -0,0 +1,39 @@
+import numcodecs
+from codecs import PngCodec, Lz4, NumPy, Zstd, JpegCodec
+
+
+def _get_compressor(compressor: str, **kwargs):
+    """Get compressor object
+
+    Example:
+        compressor = _get_compressor('lz4', acceleration=2)
+
+    Args:
+        compressor (str): lowercase name of the required compressor
```

Comment:
Could you list the supported `kwargs` here? It's hard to tell at a glance what options are supported and what their key names are